io/serialport/25-serial.js

```diff
@@ -41,6 +41,20 @@ module.exports = function(RED) {
             node.port = serialPool.get(this.serialConfig);
 
             node.on("input",function(msg) {
+				if (msg.hasOwnProperty("baudrate")) {
+					var baud = parseInt(msg.baudrate);
+					if (isNaN(baud)) {
+						var errmsg = err.toString().replace("Serialport","Serialport "+node.port.serial.path);
```

Comment:
There is no err at this point - you should just set it to the string you want. But the string needs to be the translated one - so something like `node.error(RED._("serial.errors.badbaudrate"),msg);`
and then create a badbaudrate entry in the US locale file.

Reply:
I did it, but I must admit it is over my actual knowledge.